specification/azure-kusto/resource-manager/Microsoft.Kusto/2017-09-07-privatepreview/examples/KustoOperationsList.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "parameters": {
+    "api-version": "2017-09-07-privatepreview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "operationName",
+            "display": {
+              "provider": "providerName",
+              "resource": "resourceName",
+              "operation": "operationName",
+              "description": "description"
```

Comment:
This is a false positive but the linter is complaining the value of "description" cannot be "description".

No action needed but just want to document this here.